src/__swaps__/screens/Swap/components/CoinRow.tsx

```diff
@@ -39,15 +40,8 @@ export const CoinRow = ({
   symbol: string;
 }) => {
   const theme = useTheme();
-  const { favoritesMetadata } = useFavorites();
 
-  const favorites = Object.values(favoritesMetadata);
-
-  const isFavorite = (address: string) => {
-    return favorites.find(fav =>
-      fav.address === ETH_ADDRESS ? '0x0000000000000000000000000000000000000000' === address : fav.address === address
-    );
-  };
+  const [isFavorited, setIsFavorited] = useState<boolean>(isFavorite(address));
```

Comment:
i think with the removal of the `key` prop on CoinRow this has the possibility to be wrong. 

See https://shopify.github.io/flash-list/docs/fundamentals/performant-components#remove-key-prop

Reply:
Also see https://shopify.github.io/flash-list/docs/recycling